starboard/evergreen/shared/launcher.py

```diff
@@ -277,26 +277,24 @@ def _StageTargetsAndContentsRaspi(self):
 
   def _StageTargetsAndContentsRdk(self):
     """Stage targets and their contents for GN builds for RDK platforms."""
-    content_subdir = os.path.join('usr', 'share', 'cobalt')
 
     # The rdk loader always loads libcobalt.so. Hard code target_name to match.
     target_name = 'cobalt'
 
     # Copy target content and binary.
-    target_install_path = os.path.join(self.out_directory, 'install')
     target_staging_dir = os.path.join(self.staging_directory, 'content', 'app',
                                       target_name)
     os.makedirs(target_staging_dir)
 
     # TODO(b/218889313): Reset the content path for the evergreen artifacts.
-    content_subdir = os.path.join('usr', 'share', 'cobalt')
-    target_content_src = os.path.join(target_install_path, content_subdir)
+    target_content_src = os.path.join(self.out_directory, 'content')
     target_content_dst = os.path.join(target_staging_dir, 'content')
     shutil.copytree(target_content_src, target_content_dst)
     shlib_name = f'lib{self.target_name}.so'
 
     target_binary_src = os.path.join(self.out_directory, shlib_name)
-    target_binary_dst = os.path.join(target_staging_dir, 'lib', shlib_name)
+    target_binary_dst = os.path.join(target_staging_dir, 'lib',
+                                     f'lib{target_name}.so')
```

Comment:
Why remove `shlib_name` here?

Reply:
"shlib_name" is the real target name, e.g. "libbase_unittests". RDK loader goes after "libcobalt.so" only. Therefore it has to be renamed to "libcobalt.so" on and only on RDK.

Reply:
`shlib_name` is set to `libcobalt.so` on line 296 since `target_name` is hardcoded to `cobalt` above.

Reply:
No. shlib_name is associated with self.target_name, which is not the local target_name. 
self.target_name = 'base_unittests"
target_name = 'cobalt'

Reply:
Oh, I see now. `target_name` vs `self.target_name` is pretty confusing. Can you rename `target_name` something like `rdk_loader_target`?